sycl/tools/sycl-xocc/bin/sycl-xocc

```diff
@@ -93,10 +93,19 @@ else
   # ENV variable
 
   SDX_LIB_SPIR="$SDX_BIN_PATH_DIR/../lnx64/lib/libspir64-39-hls.bc"
+  if [[ ! -f "$SDX_LIB_SPIR" ]]; then
+    SDX_LIB_SPIR="$SDX_BIN_PATH_DIR/../../../Vitis_HLS/2020.3/lnx64/lib/libspir64-39-hls.bc"
```

Comment:
This sounds super hard-coded...
```suggestion
    SDX_LIB_SPIR="$SDX_BIN_PATH_DIR"/../../../Vitis_HLS/2*/lnx64/lib/libspir64-39-hls.bc"
```
At least we are good up to year 3000. :-)
We need `2*` because on internal build it would be ambiguous with `HEAD`.